pages/common/sed.md

```diff
@@ -1,36 +1,29 @@
 # sed
 
 > Edit text in a scriptable manner.
+> See also: `awk`, `ed`.
 > More information: <https://www.gnu.org/software/sed/manual/sed.html>.
 
-- Replace the first occurrence of a regular expression in each line of a file, and print the result:
+- Execute a specific expression:
 
-`sed 's/{{regular_expression}}/{{replace}}/' {{filename}}`
+`{{command}} | sed '{{s/apple/mango/g}}'`
 
```

Comment:
I think the original was much clearer. Let's revert this change.

Reply:
Done ;)